src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -12,6 +12,22 @@
 
 <H2>Charset Mismatch</H2>
 This check identifies responses where the HTTP Content-Type header declares a charset different from the charset defined by the body of the HTML or XML. When there's a charset mismatch between the HTTP header and content body Web browsers can be forced into an undesirable content-sniffing mode to determine the content's correct character set.
+This check handles various conditions depending on the Threshold set for the scanner, as follows:
+<ul>
+  <li>All Thresholds:</li>
```

Comment:
Oh ok, so instead of a list within a list it's a line item containing a list....interesting. Will do, fix-up coming shortly.